robotoff/models.py

```diff
@@ -171,6 +171,12 @@ class Prediction(BaseModel):
     )
     predictor = peewee.CharField(max_length=100, null=True)
 
+    def crop_image_url(self, source_image) -> str:
```

Comment:
this was meant to be a module level function, you wont have a prediction but an insight in you slack log function.

You should add it at module level, with two parameters : source_image, bounding_box

Please also refactor AnnotationVote.crop_image_url to reuse this function.

Reply:
> Please also refactor AnnotationVote.crop_image_url to reuse this function.

`AnnotationVote` does not have `crop_image_url()` I am not sure which part of the code to refactor?

Reply:
Sorry it was LogoAnnotation, but you find the right one :-)